docs/configuration.md

```diff
@@ -175,6 +175,13 @@ Key | Description | Type | Default | Required
 `brands.[].keyFunctions` | List key functions available per brand ([schema][combo-keys-schema]). | `Array` | | `false`
 `brands.[].lineKeys` | List of keys to be used as default line keys per brand. | `Array` | | `false`
 
+#### `crossSiteMessaging`
```

Comment:
Please add an entry to the [Settings table above](https://github.com/2600hz/monster-ui/blob/a7eee0c7f85cd6fedce29366cddbc96c871e0d30/docs/configuration.md#settings), to indicate where the `crossSiteMessaging` should be included.